source/localizable/feedspec/index.en.html.md.erb

```diff
@@ -594,6 +596,225 @@ For example: If there are 10 available pieces of Medium size, 15 of Large and 5
+descriptions that will be crosslinked to the ones provided in your feed.
+
+Available in store / Delivery 1 to 3 days
+: refers to products that are available for pick up at your outlet.
+
+Delivery 1 to 3 days
+: refers to products that can be delivered to your customer within 1 to 3 days.
+
+Delivery 4 to 10 days
+: refers to products that can be delivered to your customer within 4 to 10 days.
+
+Delivery up to 30 days
+: refers to products that can be delivered to your customer within 30 days.
+
+##### Example
+
+~~~ xml
+<availability>1 to 3 days</availability>
+~~~
+
+
+#### Variation Size
+
+If the product comes in sizes you can include all available sizes in size variations attribute.
+
+For example, a sport shoe available in 38 your `XML` will include
+the following attribute
+
+~~~ xml
+<size>38</size>
+~~~
+
+You can also use `1/2` as a half size indicator
+
+~~~ xml
+<size>38 1/2</size>
+~~~
+
+In case of clothes, sizes can be in one of the following formats:
+
+~~~ xml
+<size>XL</size>
+~~~
+
+~~~ xml
+<size>Extra Large</size>
+~~~
+
+~~~ xml
+<size>10</size>
+~~~
+
+If a size variation fits a size range (e.g. socks) then you can declare the size as a
+*range* using the dash (-). e.g.
+
+~~~ xml
+<size>5.5-6.5</size>
+~~~
+
+Finally, if a size variation is identified by two sizes you can include them using the
+forward slash. (/) e.g.
+
+~~~ xml
+<size>5.5/42</size>
+~~~
+
+> Size variations that are identified by sizes and the size information is not provided
+will **not** be classified.
+
+
+#### Variation Quantity
+
+The quantity is the amount of items available for purchase via Skroutz.
+
+> The quantity field only accepts integer values greater or equal to `0` and less or equal to `10000000`.
+
+> If the size variation is out of stock, the indicated value should be `0`.
+
+##### Example
+
+~~~ xml
+<quantity>10</quantity>
+~~~
+
+
+#### Variation Link
+
+The URL that links to your shop's page for the specified size variation. It must be a
+valid URL starting with `https`.
+
+> Size variation links **should not** be URL-encoded.
+
+> This link must only point to a product page and not a category page or a
+collection of products page.
+
+##### Example
+
+~~~ xml
+<link>https://www.mydomain.com/products/1</link>
+<url>https://www.mydomain.com/products/product.php?itemid=1</url>
+<url><![CDATA[https://www.mydomain.com/products/product.php?itemid=1&amp;somevar=3]]></url>
+<url>https://www.mydomain.com/products/product.php?itemid=1&amp;amp;somevar=3</url>
+~~~
+
+
+#### Variation Price
+
+It's the size's variation retail price **including V.A.T**.
+
+> The price of the size variation must apply to everyone without any special conditions
+(e.g. offers, discounts pending on location etc).
+
+> The price field accepts up to 2 decimal places.
+
+##### Example
+
+~~~ xml
+<price>23.22</price>
+<pricevat>23.22</pricevat>
+~~~
+
+
+#### Variation MPN
+
+The unique manufacturer identifier of the product. It is used to identify a
```

Comment:
identifier of the variation. If it's the same as the product, why would we require it anyway?

Reply:
Oh I see, you mean the "product - item" not the "product" concept.. If nobody else finds this confusing we can keep it